addons/l10n_fi/data/res_country_group_data.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Since account.fiscal.position.template doesn't have country_id -->
+    <!-- but account.fiscal.position has, this has to be manually set for -->
+    <!-- the intra-community fiscal position. -->
+    <record id="eu_wo_finland" model="res.country.group">
```

Comment:
We have a `base.europe` group that can be used. The `account.fiscal.position` has a sequence, so you have to make sure that the position of your country has a lower sequence than the europe to be selected first.